src/CMakeLists.txt

```diff
@@ -466,6 +466,20 @@ if(WIN32)
   # The UNICODE and _UNICODE definitions are needed to make
   # (QString) mudlet::getShortPathName(const QString& name) work in Windows:
   target_compile_definitions(mudlet PRIVATE UNICODE _UNICODE)
+
+  add_custom_command(TARGET mudlet POST_BUILD
```

Comment:
I think you will need to make this conditional on `USE_MAIN_BUILD_SYSTEM` as well - for starters my MSYS2/Mingw-w64 setup has different versions of some files, including `libhunspell-1.7.0.dll` and has a `cp` rather than `copy` as the file duplication command in a native (bash-like) shell....

Reply:
... and some of the file names are also different for those building 64-bit versions (`libssl-1_1--x64.dll`, `libcrypto-1_1-x64.dll`) instead of the one listed here for 32-bit builds.